### PR TITLE
Services

### PR DESCRIPTION
**Pull request by hadiaHani**
complete profile view
complete resident profile view
complete meeting details view
services/expenses / add expenses/ ads details/edit resident/edit details/edit meeting Views => Just UI

------------------
**New** 
Resolve conversation
complete Ads Details view

### CODE REVIEW

**Review by DevHeyamAwad — APPROVED** [review]

**Review by ZainabH3 — APPROVED** [review]

File: lib/features/ads/presentation/view/add_ads_view.dart [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          children: [
+            additionBox(),
+            SizedBox(
+              height: ManagerHeight.h10,
+            ),
+            baseTextFormField(hintText: ManagerStrings.title),
+            SizedBox(
+              height: ManagerHeight.h10,
+            ),
+            baseTextFormField(hintText: ManagerStrings.details, maxLines: 6),
```
Comment (mayadaabed) [inline lib/features/ads/presentation/view/add_ads_view.dart:33]: maxLines: 6
move 6 to constants

File: lib/features/ads/presentation/view/edit_ads_view.dart [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          children: [
+            additionBox(),
+            SizedBox(
+              height: ManagerHeight.h10,
+            ),
+            baseTextFormField(hintText: ManagerStrings.title),
+            SizedBox(
+              height: ManagerHeight.h10,
+            ),
+            baseTextFormField(hintText: ManagerStrings.details, maxLines: 6),
```
Comment (mayadaabed) [inline lib/features/ads/presentation/view/edit_ads_view.dart:33]: here too

File: lib/features/meetings/presentation/view/add_meeting_view.dart [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                    hintText: ManagerStrings.hourMeeting,
+                  ),
+                ),
+              ],
+            ),
+          ),
+          SizedBox(
+            height: ManagerHeight.h10,
+          ),
+          baseTextFormField(maxLines: 5, hintText: ManagerStrings.details),
```
Comment (mayadaabed) [inline lib/features/meetings/presentation/view/add_meeting_view.dart:64]: here too

File: lib/core/network/app_api.dart [hunk 4]
```diff
@@ -81,4 +84,17 @@ abstract class AppApi {
   Future<FinalAdsResponse> getAds();
   @GET(RequestConstants.meetingsRequest)
   Future<FinalMeetingsResponse> getMeetings();
+
+  @GET(RequestConstants.showResidentRequest)
+  Future<FinalShowResidentResponse> showResident(
+    @Path("id") int id,
```
Comment (Muhanned-Anwar) [inline lib/core/network/app_api.dart:90]: put path id as a constant in constants file

File: lib/core/network/app_api.dart [hunk 5]
```diff
@@ -81,4 +84,17 @@ abstract class AppApi {
...
+  );
+
+  @GET(RequestConstants.showMeetingRequest)
+  Future<FinalShowMeetingResponse> showMeeting(
+    @Path("id") int id,
```
Comment (Muhanned-Anwar) [inline lib/core/network/app_api.dart:95]: put path id as a constant in constants file

File: lib/features/ads/presentation/view/ads_detailes_view.dart [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+              ManagerStrings.details,
+              style: getBoldStyle(
+                  fontSize: ManagerFontSize.s16,
+                  color: ManagerColors.primaryTextColor),
+            ),
+            SizedBox(
+              height: ManagerHeight.h10,
+            ),
+            Text(
+              "هذا النص هو مثال لنص يمكن أن يستبدل في نفس المساحة، لقد تم توليد هذا النص من مولد النص العربى، حيث يمكنك أن تولد مثل هذا النص أو العديد من النصوص الأخرى إضافة إلى زيادة عدد الحروف التى يولدها التطبيق",
```
Comment (Muhanned-Anwar) [inline lib/features/ads/presentation/view/ads_detailes_view.dart:58]: move to manager strings

File: lib/features/ads/presentation/view/ads_detailes_view.dart [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+import 'package:my_building_project/core/widget/main_button.dart';
+import 'package:my_building_project/routes/routes.dart';
+
+class AdsDetailesView extends StatelessWidget {
+  const AdsDetailesView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
```
Comment (Muhanned-Anwar) [inline lib/features/ads/presentation/view/ads_detailes_view.dart:18]: I prefer using custom app bar for all app

**Review by Muhanned-Anwar — CHANGES_REQUESTED** [review]

File: lib/features/ads/presentation/view/ads_detailes_view.dart (hunk 6 above)
Comment (hadiaHani) [reply lib/features/ads/presentation/view/ads_detailes_view.dart]: Just for the test. it has been replaced with a text from the API

**Review by DevHeyamAwad — APPROVED** [review]

File: lib/core/network/app_api.dart [hunk 8]
```diff
@@ -97,4 +98,9 @@ abstract class AppApi {
 
   @GET(RequestConstants.profileRequest)
   Future<ProfileResponse> getProfile();
+
+  @GET(RequestConstants.showAdsRequest)
+  Future<FinalShowAdsResponse> showAds(
+    @Path("id") int id,
```
Comment (Muhanned-Anwar) [inline lib/core/network/app_api.dart]: move to constants

**Review by Muhanned-Anwar — CHANGES_REQUESTED** [review]

**Pull request merged**